docs/_includes/primary-nav-items.html

```diff
@@ -5,6 +5,9 @@
   <li class="{% if page.url contains '/docs/' %}current{% endif %}">
     <a href="/docs/home/">Docs</a>
   </li>
+  <li class="{% if page.url contains '/tutorials/' %}current{% endif %}">
+    <a href="/tutorials/home/">Tutorials</a>
+  </li>
```

Comment:
I could remove the "Tutorials" link from the homepage nav and list it on the ["Help" page](http://jekyllrb.com/help/) right under Documentation. Would that work?